app/controllers/api/v1/tickets_controller.rb

```diff
@@ -15,13 +15,50 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 class Api::V1::TicketsController < Api::V1::ApplicationController
 
-	load_and_authorize_resource :ticket
+    load_and_authorize_resource :ticket
 
-	def index
-		@tickets = Ticket.by_status(:open).viewable_by(current_user)
-	end
+    def index
+        @tickets = Ticket.by_status(:open).viewable_by(current_user)
+    end
 
-	def show
-		@ticket = Ticket.find(params[:id])
-	end
+    def show
+        @ticket = Ticket.find(params[:id])
+    end
+
+    def create
+        @ticket = Ticket.new(ticket_params)
+        if !@ticket.nil? && @ticket.save
+          NotificationMailer.incoming_message(@ticket, params[:message])
+        end
+        render nothing: true, status: :created
+    end
+
+    protected
```

Comment:
Indent access modifiers like `protected`.<br>Keep a blank line before and after `protected`.